contracts/src/v0.8/automation/dev/test/AutomationRegistry2_3.t.sol

```diff
@@ -88,6 +96,179 @@ contract AutomationRegistry2_3_CheckUpkeep is AutomationRegistry2_3_SetUp {
   }
 }
 
+contract AutomationRegistry2_3_Withdraw is AutomationRegistry2_3_SetUp {
+  address internal aMockAddress = address(0x1111111111111111111111111111111111111113);
+
+  function getLinkBalance(address balanceAddress) public view returns (uint256) {
+    return link.balanceOf(balanceAddress);
+  }
+
+  function getNonLinkBalance(address balanceAddress) public view returns (uint256) {
+    return nonLink.balanceOf(balanceAddress);
+  }
+
+  function mintLink(address recipient, uint256 amount) public {
+    //record the current address and switch to the recipient
+    address originalAddr = msg.sender;
+    changePrank(ADMIN);
+
+    //mint the nonLink to the recipient
+    link.mint(recipient, amount);
+
+    //change back to the original address
+    changePrank(originalAddr);
+  }
+
+  function mintNonLink(address recipient, uint256 amount) public {
+    //record the current address and switch to the recipient
+    address originalAddr = msg.sender;
+    changePrank(ADMIN);
+
+    //mint the nonLink to the recipient
+    nonLink.mint(recipient, amount);
+
+    //change back to the original address
+    changePrank(originalAddr);
+  }
+
+  function setConfigForWithdraw() public {
```

Comment:
could we just outright run this function so that the registry is setup for each test?

Reply:
Hey @RyanRHall , i didnt get this one, can you elaborate? 

Here is my thought for having this function: I extracted this setConfig to this function so we the tests can share this function, and save 30 lines of code. And yes, the tests run this function at the beginning to setup.